packages/media/src/hasVideoGotAudio.ts

```diff
@@ -14,15 +14,28 @@
  * limitations under the License.
  */
 
+// See https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-9.html
+
+interface AudioTrackList {
+  [Symbol.iterator](): IterableIterator<unknown>;
+  length: number;
+}
+
+interface HTMLVideoElement {
+  readonly audioTracks?: AudioTrackList;
+  readonly mozHasAudio?: boolean;
+  readonly webkitAudioDecodedByteCount?: number;
+}
+
 /**
  * Determines whether a video element has audio tracks.
  *
  * @param {HTMLVideoElement} video Video element.
  * @return {boolean} Whether the video has audio or not.
  */
-function hasVideoGotAudio(video) {
+function hasVideoGotAudio(video: HTMLVideoElement): boolean {
   return (
-    video.mozHasAudio ||
+    (video.mozHasAudio as boolean) ||
```

Comment:
<!-- potential concern, ask for clarification or investigation -->
**❓question:**
Does this need `as boolean` even though this prop's type is configured as `boolean` above? I thought it would be required only if TypeScript wouldn't know what the type is. Thoughts?

Reply:
Indeed